publication-event-handlers/src/test/resources/doirequests/resource_update_event_new_image_only_with_doi.json

```diff
@@ -18,28 +18,55 @@
       "executedVersion": "$LATEST"
     },
     "responsePayload": {
-      "type": "publication.update",
+      "type": "doirequest.update",
       "updateType": "INSERT",
-      "newPublication": {
-        "type": "Publication",
-        "identifier": "017768664574-6236cf35-a0a9-4eaf-84ea-fbcd9d7cf413",
-        "status": "DRAFT",
-        "owner": "og@unit.no",
-        "doi": "https://10.10000/100/12",
-        "publisher": {
-          "type": "Organization",
-          "id": "https://api.dev.nva.aws.unit.no/customer/f54c8aa9-073a-46a1-8f7c-dde66c853934"
+      "newData": {
+        "type": "DoiRequest",
```

Comment:
this is of type "Resource" where the Resource has a non empty doi. (and no DoiRequest). See previous comment.

Reply:
If I change this to "Resource" I get eventType=empty. Is this the expected result while we have an active bug registration for this?